### PR TITLE
Divide into redux-api and js-api

### PR DESCRIPTION
**Pull request by vedrani**

### CODE REVIEW

**Comment by vedrani** [comment]
@zrumenjak What will we use for npm package names?
`@shoutem/js-api` - fetch javascript api sdk
`@shoutem/redux-api` - redux api sdk, depends upon `js-api`

File: fetch-api/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@shoutem/js-api",
```
Comment (zrumenjak) [inline fetch-api/package.json]: Can we rename this to `api-sdk`? `js-api` was a bit confusing to me when I first saw it. It looked like this is some sort of JS interface with other Shoutem components...

We also had a convention to to add `react-` before a package name when it was react specific, so I would call this:
```
api-sdk
react-api-sdk
```

Sorry about this, I know you asked about it a long time ago, but it seemed ok to me at that point.

File: redux-api/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "isparta": "^4.0.0",
+    "istanbul": "0.4.4",
+    "mocha": "^2.5.3",
+    "nock": "^8.2.1",
+    "rimraf": "^2.5.2",
+    "sinon": "^1.17.4"
+  },
+  "dependencies": {
+    "@shoutem/js-api": "file:../fetch-api",
+    "@shoutem/js-api": "0.1.1-beta.2",
```
Comment (zrumenjak) [inline redux-api/package.json]: You have js-api dependency twice here... Is this intentional?

**Review by zrumenjak — APPROVED** [review]

File: fetch-api/package.json (hunk 1 above)
Comment (vedrani) [reply fetch-api/package.json]: No problem. Just a question regarding `react-` suggestion. There is nothing react specific, below are dependencies, oriented toward redux. Does `react` is only meant to flag ecosystem stack?
```
 "@shoutem/js-api": "file:../fetch-api",
    "@shoutem/js-api": "0.1.1-beta.2",
    "@shoutem/redux-composers": "0.1.3",
    "@shoutem/redux-io": "2.0.2-beta.1",
    "flux-standard-action": "^1.2.0",
    "lodash": "^4.17.4",
    "urijs": "^1.18.9"
```

File: redux-api/package.json (hunk 2 above)
Comment (vedrani) [reply redux-api/package.json]: Yes, this is map, and last key overrides all the rest keys. If you want to test fetch-api in redux-api link allows you to do it quickly. Json doesnt support comments :(

**Pull request merged**